kolibri/core/analytics/tasks.py

```diff
@@ -24,7 +25,9 @@
 @register_task(job_id=DEFAULT_PING_JOB_ID)
 def _ping(started, server, checkrate):
     try:
-        ping_once(started, server=server)
+        pingback_id = ping_once(started, server=server)
+        if pingback_id:
+            ping_error_reports.enqueue(args=(server, pingback_id))
```

Comment:
I see this is creating two different pathways that hinges on the `pingback_id`. In `utils.py`, there already exists logic dependent on `if "id" in data:`, which is the same condition here. It seems like this fits alongside the existing logic there.